src/components/Game.js

```diff
@@ -52,26 +70,46 @@ const Game = () => {
 
   return (
     <div className="game">
-      <div className="game-info">
-        <h1>Tic-Tac-Toe</h1>
-        <div className="game-player">
-          <form>
-            <label>
-              Player X
-              <input type="text" onChange={changePlayerX} value={playerX} />
-            </label>
-            <label>
-              Player O
-              <input type="text" onChange={changePlayerO} value={playerO} />
-            </label>
-          </form>
+      <div className="game__info">
+        <h1 className="game__title">
+          Tic-Tac-Toe
+          <button onClick={showRules} className="button">
+            ?
+          </button>
+        </h1>
+        <div className="game__content">
+          <div className="game__player">
+            <div className="game__player--x">
```

Comment:
This was my mistake, it should be:

`game__player-x`
`game__player-o`

Unless you're going to define a new block like we discussed!